Nil pointer error appears at v0.14.0

Hello @fujiwara, thanks for the great tool.
I found a error when exec `lambda init`.

## Error
Nil pointer error appears using new version lambroll when execute `lambroll init` without exacting lambda func.

```bash
$ lambrall version
lambroll v0.14.0
$ lambroll init --function-name=testFunction
2022/11/30 01:15:06 [info] lambroll v0.14.0 with function.json
2022/11/30 01:15:08 [info] function testFunction is not found
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x198d794]

goroutine 1 [running]:
github.com/fujiwara/lambroll.newFunctionFrom(0xc00083f758, 0x0, 0x0)
        github.com/fujiwara/lambroll/lambroll.go:257 +0x214
github.com/fujiwara/lambroll.(*App).Init(0xc000091040, {0xc000511e40?, 0xc00060c60f?})
        github.com/fujiwara/lambroll/init.go:73 +0x4f5
main._main()
        github.com/fujiwara/lambroll/cmd/lambroll/main.go:146 +0x58b3
main.main()
        github.com/fujiwara/lambroll/cmd/lambroll/main.go:19 +0x19
```

But no error appears using v0.13.0.
```bash
$ lambrall version
lambroll v0.13.0
$ lambroll init --function-name=testFunction
2022/11/30 01:51:22 [info] lambroll v0.13.0 with function.json
2022/11/30 01:51:23 [info] function testFunction is not found
2022/11/30 01:51:24 [info] creating .lambdaignore
2022/11/30 01:51:24 [info] creating function.json
Overwrite existing file function.json? (y/n) [n]: n
2022/11/30 01:51:28 [info] completed
```

## Environment
lambroll version: v0.14.0
OS: macOS monterey 12.3.1

--- a/function_test.go
+++ b/function_test.go
@@ -3,6 +3,9 @@ package lambroll
 import (
 	"os"
 	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lambda"
 )
 
 func TestLoadFunction(t *testing.T) {
@@ -60,5 +63,44 @@ func TestLoadFunction(t *testing.T) {
 			t.Errorf("unexpected EphemeralStorage %v", fn.EphemeralStorage)
 		}
 		t.Log(fn.String())
+	}
+}
+
+func TestNewFunction(t *testing.T) {
+	conf := &lambda.FunctionConfiguration{
+		FunctionName: aws.String("hello"),
+		MemorySize:   aws.Int64(128),
+		Runtime:      aws.String("nodejs14.x"),
+		Timeout:      aws.Int64(3),
+		Handler:      aws.String("index.handler"),
+		Role:         aws.String("arn:aws:iam::0123456789012:role/YOUR_LAMBDA_ROLE_NAME"),
+	}
+	tags := map[string]*string{
+		"foo": aws.String("bar"),
+	}
+	fn := newFunctionFrom(conf, nil, tags)
+	if *fn.FunctionName != "hello" {
+		t.Errorf("unexpected function name got %s", *fn.FunctionName)
+	}
+	if *fn.MemorySize != 128 {
+		t.Errorf("unexpected memory size got %d", *fn.MemorySize)
+	}
+	if *fn.Runtime != "nodejs14.x" {
+		t.Errorf("unexpected runtime got %s", *fn.Runtime)
+	}
+	if *fn.Timeout != 3 {
+		t.Errorf("unexpected timeout got %d", *fn.Timeout)
+	}
+	if *fn.Handler != "index.handler" {
+		t.Errorf("unexpected handler got %s", *fn.Handler)
+	}
+	if *fn.Role != "arn:aws:iam::0123456789012:role/YOUR_LAMBDA_ROLE_NAME" {
+		t.Errorf("unexpected role got %s", *fn.Role)
+	}
+	if *fn.Tags["foo"] != "bar" {
+		t.Errorf("unexpected tags got %v", fn.Tags)
+	}
+	if fn.SnapStart != nil {
+		t.Errorf("unexpected snap start got %v", fn.SnapStart)
 	}
 }

--- a/lambroll.go
+++ b/lambroll.go
@@ -253,10 +253,9 @@ func newFunctionFrom(c *lambda.FunctionConfiguration, code *lambda.FunctionCodeL
 		DeadLetterConfig:  c.DeadLetterConfig,
 		FileSystemConfigs: c.FileSystemConfigs,
 		KMSKeyArn:         c.KMSKeyArn,
-		SnapStart: &lambda.SnapStart{
-			ApplyOn: c.SnapStart.ApplyOn,
-		},
+		SnapStart:         newSnapStart(c.SnapStart),
 	}
+
 	if e := c.Environment; e != nil {
 		fn.Environment = &lambda.Environment{
 			Variables: e.Variables,
@@ -288,6 +287,15 @@ func newFunctionFrom(c *lambda.FunctionConfiguration, code *lambda.FunctionCodeL
 	fn.Tags = tags
 
 	return fn
+}
+
+func newSnapStart(s *lambda.SnapStartResponse) *lambda.SnapStart {
+	if s == nil {
+		return nil
+	}
+	return &lambda.SnapStart{
+		ApplyOn: s.ApplyOn,
+	}
 }
 
 func exportEnvFile(file string) error {

